src/drawers/arcDrawer.ts

```diff
@@ -6,6 +6,7 @@ export module Drawers {
 
     constructor(dataset: Dataset) {
       super(dataset);
+      this._className = "arc";
```

Comment:
The class name is added here to specify what types of paths an Arc Drawer will draw.  Note that Symbol Drawer labels with a "symbol" class